examples/self_supervised/Dockerfile

```
@@ -0,0 +1,6 @@
+FROM python:3.8
+
+RUN pip install catalyst[cv]==21.09
+RUN pip install catalyst[ml]==21.09
```

```suggestion
RUN pip install catalyst[cv]==21.09 catalyst[ml]==21.09
```
is it possible to do so?